src/software/geom/util_test.cpp

```diff
@@ -1197,36 +1131,6 @@ TEST(GeomUtilTest, test_find_open_circle_points_outside_of_box)
     ASSERT_EQ(0, empty_circles.size());
 }
 
-TEST(GeomUtilTest, test_point_polygon_dist_point_contained_in_polygon)
-{
-    Polygon polygon = Polygon({Point(0, 0), Point(1, 0), Point(1, 1), Point(0, 1)});
-    Point point     = Point(0.5, 0.5);
-
-    double result = dist(point, polygon);
-
-    EXPECT_DOUBLE_EQ(0.0, result);
-}
-
-TEST(GeomUtilTest, test_point_polygon_dist_point_close_to_polygon)
-{
-    Polygon polygon = Polygon({Point(0, 0), Point(1, 0), Point(1, 1), Point(0, 1)});
-    Point point     = Point(0.5, 1.1);
-
-    double result = dist(point, polygon);
-
-    EXPECT_NEAR(0.1, result, 1e-9);
-}
-
-TEST(GeomUtilTest, test_point_polygon_dist_point_far_from_polygon)
-{
-    Polygon polygon = Polygon({Point(0, 0), Point(1, 0), Point(1, 1), Point(0, 1)});
-    Point point     = Point(-5, -5);
-
-    double result = dist(point, polygon);
-
-    EXPECT_DOUBLE_EQ(std::hypot(-5, -5), result);
-}
```

Comment:
Do we have equivalents to these tests for `new_geom`?

Reply:
Yes for the most part. In `distance_test` we have: `point_in_polygon`, `point_on_polygon_edge`, `point_on_polygon_vertex`, `point_off_polygon_closest_to_vertex`, `point_near_polygon_closest_to_edge`, `point_far_from_polygon`, `point_in_rectangle`, `point_near_rectangle`. Some of the polygon tests are actually with rectangles, but these tests don't explicitly consider cases of point being above/below/left/right of rectangle.

The tests on `dist(Segment, Point)` in `distance_test` cover the test in `util_test`.

Reply:
Done